cmd/replicate-add.go

```diff
@@ -135,19 +149,37 @@ func mainReplicateAdd(cliCtx *cli.Context) error {
 	fatalIf(err, "Unable to initialize connection.")
 	rcfg, err := client.GetReplication(ctx)
 	fatalIf(err.Trace(args...), "Unable to get replication configuration")
-	ruleStatus := "enable"
-	if cliCtx.Bool("disable") {
-		ruleStatus = "disable"
+	ruleStatus := enableStatus
+	if cliCtx.Bool(disableStatus) {
+		ruleStatus = disableStatus
+	}
+	dmReplicateStatus := disableStatus
+	deleteReplicationStatus := disableStatus
+	if cliCtx.IsSet("replicate") {
+		replSlice := strings.Split(cliCtx.String("replicate"), ",")
+		for _, opt := range replSlice {
+			switch strings.TrimSpace(strings.ToLower(opt)) {
+			case "delete-marker":
+				dmReplicateStatus = enableStatus
+			case "delete":
+				deleteReplicationStatus = enableStatus
+			default:
+				fatalIf(probe.NewError(fmt.Errorf("invalid value for --replicate flag %s", cliCtx.String("replicate"))), "--replicate flag takes one or more comma separated string with values \"delete, delete-marker\"")
```

Comment:
For completeness purposes, I guess we can explain about the 3rd valid value, `""`, which also sets the delete-marker replication to `"Disabled"`.